### PR TITLE
Fix error in nightly spark: add env PYSPARK

### PR DESCRIPTION
**Pull request by miguelgfierro**
### Description

Fix the error in the nightly build in the benchmark notebook for Spark

### Related Issues

Fix https://github.com/microsoft/recommenders/issues/1898

### Checklist:

- [ ] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [ ] I have added tests covering my contributions.
- [ ] I have updated the documentation accordingly.
- [ ] This PR is being made to `staging branch` and not to `main branch`.

### CODE REVIEW

**Comment by miguelgfierro** [comment]
Nightly working: https://github.com/microsoft/recommenders/actions/runs/4563686982/jobs/8052496808

Kudos to @loomlike

File: tests/ci/azureml_tests/test_groups.py [hunk 1]
```diff
@@ -146,8 +147,7 @@
         #
         "tests/smoke/examples/test_notebooks_pyspark.py::test_als_pyspark_smoke",  # 49.53s
         "tests/integration/examples/test_notebooks_pyspark.py::test_als_pyspark_integration",  # 110.58s
```
Comment (loomlike) [inline tests/ci/azureml_tests/test_groups.py:149]: Have other spark tests been passed with AzureML while the benchmark one failed? interesting....

**Review by loomlike — APPROVED** [review]
LGTM. Thank you!

File: tests/ci/azureml_tests/test_groups.py (hunk 1 above)
Comment (miguelgfierro) [reply tests/ci/azureml_tests/test_groups.py:149]: yeah, honestly, maybe it spark is an early evil AGI or something lol

**Pull request merged**